css/style.css

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
Oh I see.  Personally I'd use LESS all the way, I don't see that the CDN is any quicker because you can use a file watcher (e.g. gulp) to build the CSS every time the LESS file changes.  I also see that changing this CSS filei back into LESS could be error prone and time consuming.  But, I'm happy to merge this as a CSS file for now.